maxim/tests/test_llamaindex.py

```
@@ -0,0 +1,159 @@
+import unittest
+import os
+import dotenv
+import asyncio
+
+from llama_index.core.tools import BaseTool
```

Ruff (0.12.2)

`llama_index.core.tools.BaseTool` imported but unused

Remove unused import: `llama_index.core.tools.BaseTool`

(F401)